docs/standard/linq/creating-xml-trees.md

```diff
@@ -0,0 +1,425 @@
+---
+title: "Creating XML Trees - .NET"
+ms.date: 12/01/2019
+ms.assetid: cc74234a-0bac-4327-9c8c-5a2ead15b595
+---
+
+# Creating XML trees (C# and Visual Basic)
```

Comment:
Did we have a VB version of this article as well?

Reply:
Yes, but the title of the article is **Introduction to XML Literals in Visual Basic** (with some similar content in **Cloning vs. Attaching (Visual Basic**). In my design document I said that I would use these two VB documents for the content of Creating XML Trees.